### PR TITLE
added moderator badges for our staff in the forums

### PR DESCRIPTION
**Pull request by chips-gpx**
Creating a draft pr bc this still isn't 100% and I need some help. 

What it does: Gives a 'Support Staff' badge to Ashli, me, Robyn, and Nate when they post to the community forum. 

What it needs to do before complete:
1. use unique user id instead of names to apply the moderator tag
2. add Andrew, Erik, Torrey, (maybe Corey) 
3. enable the js to allow for more than one class of mod (ie, Staff Support, Support Manager, CEO)

### CODE REVIEW

**Comment by chips-gpx** [comment]
https://trello.com/c/tTHyBGn9/258-add-forum-tags

**Comment by ashlibaldwin** [comment]
1. you can find user ids for people in zendesk profile urls. here's yours:
https://gaiagpshelp.zendesk.com/agent/users/362154083768/assigned_tickets
You are 362154083768

2. is there a way to automatically give agents the staff badge so that we don't have to hard code the user id everytime we hire someone?

3. Do you have a sandbox setup where I can view this so far?

**Comment by chips-gpx** [comment]
2. This is actually how it's already set up. 

3. gaiagpshelp1556318683.zendesk.com

**Comment by chips-gpx** [comment]
Do we _need_ more than one kind of mod badge? 
- bc if we don't, it's much simpler. What I have built here will work, and I can even parse it down more. 
- if we do, we will need to hard code each new agent into the js. 
-- but this would also allow us to have "support staff, support manager, CEO, developer, etc."

File: script.js [hunk 1]
```diff
@@ -129,4 +129,5 @@ $(document).ready(function() {
     var isExpanded = this.getAttribute("aria-expanded") === "true";
     this.setAttribute("aria-expanded", !isExpanded);
   });
+
```
Comment (ashlibaldwin) [inline script.js]: remove blank line? this file should have no changes

File: style.css [hunk 2]
```diff
@@ -4319,5 +4319,23 @@ ul {
     content: "\e080";    /* adjust as needed, taken from bootstrap.css */
 }
 
+.moderator:after, .support-manager:after {
```
Comment (ashlibaldwin) [inline style.css]: AFAICT `support-manager` is not being used anywhere

File: style.css [hunk 3]
```diff
@@ -4319,5 +4319,23 @@ ul {
...
+
+content: "Support Staff";
```
Comment (ashlibaldwin) [inline style.css]: Lets make this says 'Gaia GPS Staff' or 'Gaia GPS Support'

File: style.css [hunk 4]
```diff
@@ -4319,5 +4319,23 @@ ul {
...
+
+background-color: #1eb848;
+
```
Comment (ashlibaldwin) [inline style.css]: you don't need a blank line after every attribute - lets remove those

**Review by ashlibaldwin — CHANGES_REQUESTED** [review]
please see requested changes

File: script.js (hunk 1 above)
Comment (ashlibaldwin) [reply script.js]: The blank line is still showing here. you can view the diff on your PR here https://github.com/trailbehind/NG_Help_Center/pull/6/files

**Review by ashlibaldwin — APPROVED** [review]

**Pull request merged**